### PR TITLE
Capstone project for module one

### PR DESCRIPTION
**Pull request by yasinabdmahmood**
### Project implementation

- The content has been personalized and is different from the original design.
-  This [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) is used including:
  - Colors.
  - Typography: font face, size, and weight ( [Lato](https://www.latofonts.com/) is used as it is a free font similar to the one used in the original design).
  - Layout: composition and space between elements are designed to match the original design as much as possible.
- The page looks almost identical to the original design. Small adjustments like text or image changes are implemented.

- Only these 2 pages have been built:
  - The *home page*.
  - The *about page*.
- Each of these pages has two versions for 2 different screen sizes: 
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.

- Interactions
  - Links
    - The *home page* hasa link in the menu to the *about page*.
    - The logo in the header links to the *home page*.
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    - The mobile menu has a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" is created dynamically in JavaScript.
  -  JavaScript variable has been used with the data about the speakers and used when the page loads to create the HTML for this section dynamically.
  - some transitions and animation are added to different elements on the page
  -  An extra button is added to the desktop design for scrolling up automatically

[Here is the link to the Video with a brief description of the project](https://drive.google.com/file/d/1xYqlXKJc1CKMIzeIZ69recqs94L5uMrO/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project
+
+> In this website you will learn how to create a web application using the Html Css and Javascript.
```
Comment (tufoinnkuo10) [inline README.md]: - [ ] kindly add a good description of what your project is all about.  From the README the user will know what to expect

File: about.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                        <a href="#">News</a>
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">Sponser</a>
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">Join</a>
+                    </li>
+                    <li class="navbar-item-white">
+                        <a href="#">Program</a>
```
Comment (tufoinnkuo10) [inline about.html]: - [ optional] you can add links to different page sections so as to ease navigation

File: index.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Home page</title>
+</head>
+<body>
+    <section class="sec1">
+        <div class="header">
+            <button class="btn1">
```
Comment (tufoinnkuo10) [inline index.html:13]: - [x] great job with the mobile menu. Please kindly ensure that it covers the whole page when it pops-up.
Screenshot (278)_LI

File: index.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+                    </li>
+                    <li class="navbar-item-white ">
+                        <a href="./about.html">About</a> 
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="div1">
+            <span class="hello-text">"Hello, Sharing world"</span>
+            <h1 class="main-topic-text" >SCIENCE AND
```
Comment (tufoinnkuo10) [inline index.html:80]: - [ ] kindly make the h1 heading bolder or add the font weight so it looks like the Behance design

File: main.js [hunk 5]
```diff
@@ -0,0 +1,120 @@
+import createMobileMenu from './mobile_menu.js';
+import implementScrollUp from './scroll_up.js';
+
+/// ///////////////feature  speackers  section dynamic implementation ///////////
+
+const speackersInfo = [
+  {
+    name1: 'yohai benkler',
+    name2: 'Yochai Benkler',
+    img: './img/speaker_01.png',
```
Comment (tufoinnkuo10) [inline main.js:10]: - [x] add a little bit of margin to the images so the realy follow the behance design.. great work
your design 👇🏾 
Screenshot (280)_LI
behance design 👇🏾 
Screenshot (281)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @yasinabdmahmood 

Great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ design well followed

## Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success
Hey @yasinabdmahmood  :wave:

Great work on implementing the requirements. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:
:heavy_check_mark: Followed Linters
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**